### PR TITLE
feat(VEOHeader): Add Timezone switcher to the header

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes https://github.com/influxdata/ui/issues/5902

https://user-images.githubusercontent.com/18511823/192399956-d8a93e53-f927-4f96-930a-adc05d5be54c.mov

Added it to the same place that the Data Explorer has it at. 

### Checklist

Authors and Reviewer(s), please verify the following:

- [ ] A PR description, regardless of the triviality of this change, that communicates the value of this PR
- [ ] [Well-formatted conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/) that provide context into the change
- [ ] Documentation updated or issue created (provide link to issue/PR)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [ ] Feature flagged, if applicable

### CODE REVIEW

File: src/dashboards/components/VEO.scss [hunk 1]
```diff
@@ -25,6 +25,8 @@
   align-items: stretch;
   width: 100%;
   flex: 1 0 0;
+  padding-left: $cf-space-l;
+  padding-right: $cf-space-l;
```
Comment (ChitlangeSahas) [inline src/dashboards/components/VEO.scss:29]: Screen Shot 2022-09-26 at 4 49 53 PM

It had weird padding issue.

**Review by TCL735 — APPROVED** [review]

**Comment by TCL735** [comment]
Seems like editing the time zone in one cell will edit the time zone for all cells.

**Comment by ChitlangeSahas** [comment]
@TCL735 : Correct, Amy confirmed that this behavior is okay 👍

The time range selector (duration) does the same thing, adjusts the time range for all cells.

**Comment by ALuckette** [comment]
Sorry, by "cell level" I meant that we should allow users to pick unique time zones per cell. I should've been more clear, my bad. I don't think we have to roll back, but, we should make this cell-specific in a future sprint. @ChitlangeSahas

**Pull request merged**